config/initializers/filter_parameter_logging.rb

```diff
@@ -3,4 +3,4 @@
 # Be sure to restart your server when you modify this file.
 
 # Configure sensitive parameters which will be filtered from the log file.
-Rails.application.config.filter_parameters += [:password]
+Rails.application.config.filter_parameters += %i[password email name first_name middle_name last_name nhs_number nhs_letter date_of_birth carry_supplies live_in_england phone_number_calls phone_number_texts know_nhs_number basic_care_needs essential_supplies medical_conditions dietary_requirements building_and_street_line_1 building_and_street_line_2 town_city county postcode]
```

Comment:
Is this supposed to filter out everything or just personally identifiable information? For example, do the boolean values need to be filtered out?

Reply:
Hmm, probably not, but I didn't see I reason to leave it in. I was leaning towards "better safe than sorry".  But happy to remove some of them if you think it'd help as extra information in the logs.

Reply:
To be honest I'm not sure if having the information would help or not.
The only concern I really have is with keeping this list up to date.

Reply:
yeah - had this concern too. Was thinking if there was a way to not log parameters at all, or how to add test to catch new parameters.

Reply:
Yeah, I agree - this feels like it could lead to a dangerous game of whackamole where we add new questions and forget to ignore them.

Reply:
does ruby code in initalizers have access to code in controller modules?

No chance we could do something like pull out the check your answers page and map out all the possible options?

That way we'd never to whack-a-mole?

Reply:
One possible option is to use the [lograge gem][1] - that will remove the params from the log line by default and you can customise the config to add in information that you need

[1]: https://github.com/roidrage/lograge

Reply:
Yeah! I looked at adding lograge - https://github.com/alphagov/govuk-coronavirus-vulnerable-people-form/pull/214